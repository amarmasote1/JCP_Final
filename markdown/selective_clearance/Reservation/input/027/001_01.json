[  
   
   {
      "gtin" : "",
      "uom" : "EACH",
      "itemID" : "34640001027",
      "bopus" : "true",
      "lotNumber" : "3464000",
      "shippable" : "true",
      "productLine" : "346",
      "purchaseType" : "Fashion",
      "isSubstitutionAllowed" : "N",
      "itemProcessingTime" : "10"
   }
]