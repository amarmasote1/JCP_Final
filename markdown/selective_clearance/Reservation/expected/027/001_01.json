{
    "body": [
        {
            "warnings": [],
            "message": null,
            "body": null,
            "key": "JCP|34640001027|EACH",
            "status": 0
        }
    ],
    "message": "Successful",
    "status": 221
}